src/main/java/org/jabref/model/entry/event/EntriesEvent.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.model.entry.event;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.jabref.model.database.event.BibDatabaseContextChangedEvent;
+import org.jabref.model.entry.BibEntry;
+
+/**
+ * This abstract class pretends a minimal set of attributes and methods
+ * which an entries event should have.
+ */
+public abstract class EntriesEvent extends BibDatabaseContextChangedEvent {
+
+    private final List<BibEntry> bibEntries;
+    private final EntriesEventSource location;
+
+
+    /**
+     * @param List of bibEntry BibEntry objects which are involved in this event
+     */
+    public EntriesEvent(List<BibEntry> bibEntries) {
+        this(bibEntries, EntriesEventSource.LOCAL);
+    }
+
+    /**
+     * @param List of bibEntry BibEntry objects which are involved in this event
+     * @param location Location affected by this event
+     */
+    public EntriesEvent(List<BibEntry> bibEntries, EntriesEventSource location) {
+        this.bibEntries = Objects.requireNonNull(bibEntries);
+        this.location = Objects.requireNonNull(location);
+    }
+
+    //Temporary fix, while we change to plural entries
```

Comment:
Can this be removed now?

Reply:
Not yet. There are insert entry listeners that I will change later that still expect this method to be available.

Reply:
Not yet - these methods are still needed for `EntryAddedEvent` and `EntryChangedEvent` because these are still expecting the methods from the prior `EntryEvent` which I am refactoring. For `EntryChangedEvent`, which cannot refer to more than one entry, I might add the method `getBibEntry` there that works the same as the current method in `EntriesEvent`.